modules/monitoring-basic-platform-monitoring-configuration-scenario.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-platform-monitoring-basic-configuration-scenario_{context}"]
+= Platform monitoring: Basic configuration scenario
+
+This scenario describes setup steps to help administrators get started with monitoring services in a {product-title} cluster using the in-cluster monitoring stack.
+These setup steps include configuring ServiceMonitors, metrics, alerts, and persistent storage.
+
+.Prerequisites
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created users.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Set permissions for users:
+
+.. Allow users to create alerts:
++
+[source,terminal]
+----
+$ oc policy add-role-to-user monitoring-edit <user> -n <namespace>
+----
++
+.. Allow users to route alerts:
++
+[source,terminal]
+----
+$ oc -n <namespace> adm policy add-role-to-user alert-routing-edit <user>
+----
++
+. If it does not exist, create the `cluster-monitoring-config` `ConfigMap` object.
+
+.. Verify whether or not the `cluster-monitoring-config` `ConfigMap` object already exists:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring get configmap cluster-monitoring-config
+----
++
+.. If the `cluster-monitoring-config` `ConfigMap` object does not exist, create the following YAML manifest. In this example, the file is called `cluster-monitoring-config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+----
++
+.. Apply the configuration to create the `ConfigMap` object:
++
+[source,terminal]
+----
+$ oc apply -f cluster-monitoring-config.yaml
+----
+
+. Customize Prometheus to specify which metrics targets to scrape, how much and how long data will be retained, and so forth.
```

Comment:
Probably outside of the scope for this document but...Do you plan to handle additional examples for these just in the same way as for the ones above?

Reply:
Yes.

Reply:
> Probably outside of the scope for this document but...Do you plan to handle additional examples for these just in the same way as for the ones above?

Yes. Right now just making sure I'm including the right steps before spending too much time on specific instructions.